src/pages/Groups/Group4/styles/Screen1.styled.ts

```diff
@@ -0,0 +1,134 @@
+import styled from 'styled-components'
+
+export const ContainerDiv = styled.div`
+  background-color: '#ffffff';
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
+export const LoginContainer = styled.div`
+  height: 100vh;
+  width: 90vw;
+  margin: 0px 23px;
+  padding-top: 50px;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
+export const InputTextLabel = styled.text`
+  float: left;
+  font-size: 17px;
+  padding: 0px 0px 4px 0px;
+  font-family: Inter;
+  font-style: normal;
+  font-weight: 200;
+  margin-right: auto;
+`
+
+export const Logo = styled.img`
+  width: 70%;
+  max-width: 255px;
+  max-height: 255px;
+`
+export const ForgetPwButton = styled.div`
```

Comment:
button might want to use `cursor: pointer` instead since now it shows the text cursor